src/core/positioning/nmeagnssreceiver.cpp

```diff
@@ -45,6 +45,27 @@ void NmeaGnssReceiver::stateChanged( const QgsGpsInformation &info )
   if ( info.utcTime != mLastGnssPositionUtcTime )
   {
     mLastGnssPositionUtcTime = info.utcTime;
+
+    if ( mImuPosition.valid )
```

Comment:
I'd avoid using the IMU position if parseError is true.

Reply:
I removed again the mImuPosition.parseError. In fact the valid flag is enough. I have added a warning trace in case of parse errors.